examples/eks-cluster-with-observability/data.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_region" "current" {}
```

Comment:
Probably kill this file and move everything to `main.tf`. Or separate data, providers, variables and outputs out of `main.tf`.